src/lib/REST/Input/Parser/ContentTree/LoadExtendedNodeInfoRequest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\REST\Input\Parser\ContentTree;
+
+use Ibexa\AdminUi\REST\Value\ContentTree\LoadNodeExtendedInfoRequest as LoadNodeExtendedInfoRequestValue;
+use Ibexa\Contracts\Rest\Exceptions;
+use Ibexa\Contracts\Rest\Input\ParsingDispatcher;
+use Ibexa\Rest\Input\BaseParser;
+
+class LoadExtendedNodeInfoRequest extends BaseParser
+{
+    /**
+     * @param array<mixed> $data
+     */
+    public function parse(array $data, ParsingDispatcher $parsingDispatcher): LoadNodeExtendedInfoRequestValue
+    {
+        if (!array_key_exists('locationId', $data) || !is_int($data['locationId'])) {
```

Comment:
There's something about thatpart that makes me wonder - what is XML equivalent of this REST input structure?